WalletLibBitcoin/Address/BitcoinPrivateKeyAddress.swift

```diff
@@ -60,31 +60,33 @@ public class BitcoinPrivateKeyAddress {
     /// - Parameter wif: WIF string
     /// - Throws: BitcoinCreateAddressError.invalidWIFAddressLength, BitcoinCreateAddressError.invalidDataLength, BitcoinCreateAddressError.invalidWIFAddressVersion
     public init(wif: String) throws {
-        
         let wifDecoded = wif.base58decode(usingChecksum: true)
         
         guard wifDecoded.count > 3 else {
-            
             throw BitcoinCreateAddressError.invalidWIFAddressLength
-            
         }
         
-        let privateKeyData = wifDecoded.subdata(in: 1..<(wifDecoded.count - 1))
+        let versionByte = wifDecoded[0]
         
-        let privateKeyDataLength = privateKeyData.count
+        guard versionByte == BitcoinVersionByteConstant.private_key else {
+            throw BitcoinCreateAddressError.invalidWIFAddressVersion
+        }
         
-        guard privateKeyDataLength == BitcoinAddressConstants.privateKeyDataLength else {
-            
-            throw BitcoinCreateAddressError.invalidDataLength
-            
+        var privateKeyData = wifDecoded.subdata(in: 1..<(wifDecoded.count - 1))
+        
+        var wif = wif
+        if wifDecoded.count == 33 {
```

Comment:
What does the number 33 mean? I suggest avoiding magic numbers by defining them as constants with clear names, thereby adding a bit of documentation to the logic.